### PR TITLE
Ccloud profiles

### PR DESCRIPTION
**Pull request by cjmatta**
Added a `ccloud` set of templates for roles that connect brokers, also includes example inventory with the variables, and playbook.

### CODE REVIEW

**Comment by cotedm** [comment]
@cjmatta I ruined you a bit here by merging something that had been outstanding, can you rebase to 5.0.x and fix the conflicts? Then I will get on reviewing ASAP.

**Comment by gwenshap** [comment]
Hey @cjmatta, I tested the control-center script, and it looks like you included the properties template, but didn't include the "task" with the right security mode, so nothing special gets done for Cloud.

File: roles/confluent.common/defaults/main.yml [hunk 1]
```diff
@@ -4,6 +4,9 @@ zookeeper:
 broker:
   config:
     port: 9092
+schemaregistry:
+  config:
+    port: 8081
```
Comment (cricket007) [inline roles/confluent.common/defaults/main.yml]: Note: See #54 -- `schema_registry_listener_port`

**Comment by cjmatta** [comment]
Is this PR still valid, or are there other efforts here? @jcustenborder you said you were working on roles?

**Comment by JumaX** [comment]
@cjmatta @jcustenborder we've just updated with a new release which includes a variety of security options across all components.  I would recommend reviewing that.  I suspect this PR is no longer valid based on this work.  I will leave it open for a few days, and you can rebase to 5.3.0-post if you feel otherwise.

**Pull request closed without merging**